week_1/Project/youtube-fe/youtube.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube</title>
+    <link rel="stylesheet" href="styles/general.css">
+    <link rel="stylesheet" href="styles/header.css">
+    <link rel="stylesheet" href="styles/sidebar.css">
+    <link rel="stylesheet" href="styles/video.css">
+</head>
+
+<body>
+    <div class="header">
+        <div class="left-section">
+            <img class="hamburger-menu" src="icons/hamburger-menu.svg" alt="hamburger">
+            <img class="youtube-logo" src="icons/youtube-logo.svg" alt="youtube">
+        </div>
+        <div class="middle-section">
+            <input class="search-bar" type="text" placeholder="Search">
+            <button class="search-button">
+                <img class="search-icon" src="icons/search.svg" alt="search">
+            </button>
+            <button class="voice-search-button"><img class="voice-icon" src="icons/voice-search-icon.svg"
+                    alt="voice"></button>
+        </div>
+        <div class="right-section">
+            <img class="upload-icon" src="icons/upload.svg" alt="upload">
+            <img class="youtube-apps-icon" src="icons/youtube-apps.svg" alt="youtube-apps">
+            <div class="notifications-icon-container">
+                <img class="notifications-icon" src="icons/notifications.svg" alt="notification">
+                <div class="notifications-count">3</div>
+            </div>
+
+            <img class="current-user-icon" src="channel-images/colaco-channel.jpg" alt="colaco-channel">
```

Comment:
@codal-kcolaco Please check how you can lazy load image and provide hight and width to it?

Reply:
@abhi-codal Yes will do today only.